### PR TITLE
Intercom support

### PR DESCRIPTION
**Pull request by benarent**
This PR adds both intercom support and Google Analytics tracking to our docs page. 

image

### CODE REVIEW

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -98,5 +98,6 @@ under the License.
         <% end %>
       </div>
     </div>
-  </body>
```
Comment (zefer) [inline source/layouts/layout.erb]: Oops, looks like you accidentally removed the closing body tag. It should come after these partials.

File: source/layouts/layout.erb [hunk 2]
```diff
@@ -98,5 +98,6 @@ under the License.
...
+    <%= partial "layouts/intercom" %>
```
Comment (vmihailenco) [inline source/layouts/layout.erb]: Mis-indented?

**Comment by vmihailenco** [comment]
LGTM, but please fix @zefer comment

**Comment by benarent** [comment]
Thanks @vmihailenco @zefer . Merging in.

**Pull request merged**